resources/assets/app.scss

```diff
@@ -24,3 +24,41 @@ $forge-path: "~@dosomething/forge/assets/";
 .aligned-actions li {
   float: left;
 }
+
+.button.-accepted {
+  background: #80E85D;
+
+  &:hover {
+    background: lighten(#80E85D, $color-tint);
+  }
+
+  &:active {
+    background: darken(#80E85D, $color-tint);
+  }
+}
+
+.button.-rejected {
+  background: $error-color;
+
+  &:hover {
+    background: lighten($error-color, $color-tint);
+  }
+
+  &:active {
+    background: darken($error-color, $color-tint);
+  }
+}
+
+.container__block.-third {
+  @include media($tablet) {
+    @include span(4 of 12);
+  }
+}
+
+.quantity {
```

Comment:
@DFurnes hmmm I added it but when I uncheck and check it in dev tools I don't see any difference 🤔